draft-pda-protocol.md

```diff
@@ -446,29 +448,29 @@ task_id = SHA-256(param)
 Where param is just the serialization of Param.
 
 
-## Uploading Reports
+## Uploading Reports {#uploading-reports}
```

Comment:
FYI, you don't need this. kramdown automatically creates anchors by doing tr/A-Z /a-z\-/.